tools/go-cli/build.gradle

```diff
@@ -0,0 +1,75 @@
+ext.dockerImageName = "whisk/go-cli-test"
+ext.dockerContainerName = "go-cli-test"
+apply from: "../../docker.gradle"
+
+ant.condition(property: "family", value: "linux") {
+    os(family: "unix")
+}
+
+ant.condition(property: "family", value: "mac") {
+    os(family: "mac")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "amd64")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "x86_64")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "i386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "x86_32")
+}
```

Comment:
@dubeejw This is an open source project, time if infinite 😄 
I did a quick look at the LICENSE for the dependency and it's Apache 2.0, so it should be fine and compatible with this open source project.